snitun/client/connector.py

```diff
@@ -89,6 +89,8 @@ async def handler(
             # Process stream from multiplexer
             while not writer.transport.is_closing():
                 if not from_endpoint:
+                    # TODO: when paused make from_endpoint an asyncio.Future[None] that
```

Comment:
needs the same as https://github.com/NabuCasa/snitun/pull/358/files#diff-2bc5c038515962d8cda409bf8d79f098df69835d998a9150c80650f4326ee25eR126

but better to make it an object first